microsetta_public_api/repo/_alpha_repo.py

```diff
@@ -0,0 +1,27 @@
+class AlphaRepo:
+
+    def get_alpha_diversity(self, sample_ids, metric):
+        """
```

Comment:
Possible to have a one-liner here?

Reply:
https://github.com/biocore/microsetta-public-api/pull/3/commits/04cd91c78eb2bba0747e14b68019b49737792bf3